### ISSUE TITLE
Incorrect sleeping bag rendering on a placed backpack with Iris

### ISSUE THREAD

**Issue by SplendidAlakey** [Tiviacz1337/Travelers-Backpack#520]
**Describe the bug**

Sleeping bags' colours render incorrectly on placed backpacks, when Iris is installed.

**To Reproduce**

1. Install Traveler's Backpack, [Iris](https://www.curseforge.com/minecraft/mc-mods/irisshaders) and [Indium](https://www.curseforge.com/minecraft/mc-mods/indium)
2. Place down any backpack and look at its sleeping bag colour

**Expected behavior**

![2022-12-05_12 40 24](https://user-images.githubusercontent.com/86702831/205632140-4896bd28-215d-410f-8142-2e64019360bc.png)

**Screenshots**

![2022-12-05_12 46 05](https://user-images.githubusercontent.com/86702831/205632156-82b8c938-5bc6-42c9-bc8c-ae70c80a5534.png)

**Required informations (please complete the following information):**

- Mod version: 8.2.8
- Mod Loader: Fabric/Quilt

**Additional context:**

The issue seems to have appeared after the new update, that added coloured sleeping bags.

Logs: https://gist.github.com/SplendidAlakey/5ee80b9ece3845b83127e5a8debc45e8

**Comment by Tiviacz1337** [comment]
does it render correctly with tb installed only?

**Comment by SplendidAlakey** [comment]
Yes. The upper screenshot was taken with just TB, the bottom one with TB + Iris + Indium.

**Comment by Tiviacz1337** [comment]
is it possible for  you to test the file I'll send on discord?

**Comment by SplendidAlakey** [comment]
Ok, SplendidAlakey#5095. But just an FYI, I might take longer to respond on Discord, because I don't use it that often.

**Comment by Tiviacz1337** [comment]
I can't add you, send me request Tiviacz1337#1878

**Comment by Tiviacz1337** [comment]
https://www.curseforge.com/minecraft/mc-mods/travelers-backpack-fabric/files
fixed
